pkg/controller-gen/main.go

```diff
@@ -265,6 +276,56 @@ func generateClientset(groups map[string]bool, customArgs *cgargs.CustomArgs) er
 	)
 }
 
+func generateOpenAPI(groups map[string]bool, customArgs *cgargs.CustomArgs) error {
+	if len(groups) == 0 {
+		return nil
+	}
+
+	openAPIArgs := oaargs.New()
+	openAPIArgs.OutputDir = filepath.Join(customArgs.OutputBase, customArgs.Options.OutputPackage, "openapi")
+	openAPIArgs.OutputFile = "zz_generated_openapi.go"
+	openAPIArgs.OutputPkg = customArgs.Options.OutputPackage
```

Comment:
```suggestion
	openAPIArgs.OutputPkg = customArgs.Options.OutputPackage+"/openapi"
```

Otherwise the generated package name `package <name>` will be different than `openapi`.

Reply:
Ok, thank you!